packages/core/copy-uuid/src/components/CopyUuid.cy.ts

```diff
@@ -68,11 +68,49 @@ describe('<CopyUuid />', () => {
     cy.get(container).find('.uuid-icon').should('be.visible')
   })
 
-  it('renders with `isHidden` set to true', () => {
+  it('renders with `format` set to `hidden`', () => {
     cy.mount(CopyUuid, {
       props: {
         uuid,
-        isHidden: true,
+        format: 'hidden',
+      },
+    })
+
+    cy.get(container).should('be.visible')
+
+    cy.get(container).find('.uuid-container')
+      .should('have.class', 'truncated-uuid')
+      .should('have.class', 'mono')
+      .should('contain.text', '**********')
```

Comment:
Why does `hidden` look like it does the same thing as `redacted`?

Reply:
fixing that in the next push

Reply:
fixed